app/controllers/demands_controller.rb

```diff
@@ -9,7 +9,7 @@ def new
   def create
     @demand = current_user.demands.build
     @demand.assign_attributes(create_params)
-    @demand.area = current_user.area
+    @demand.area = @demand.is_statewide ? current_user.municipality.state : current_user.municipality
```

Comment:
Can we do this in the model? Or, even better, a `FormObject`? Don't know if I've added any yet but here's a quick blog post about them: https://robots.thoughtbot.com/activemodel-form-objects. I think it's worth it to change this.

Reply:
This is logic that should be run whenever a Demand is saved. If you ever save one from somewhere else you’ll have to add this logic there too.

Lemme take a swag at a form object here. Feel free to commit and I’ll refactor it?

Reply:
Ah true, okay. But yeah sure. I'm going to push some changes again shortly (not related to this comment) then we can merge and refactor this after.